stubs/python-xlib/Xlib/display.pyi

```diff
@@ -0,0 +1,177 @@
+        onerror: ErrorHandler[object] | None = ...,
+    ) -> None: ...
+    def set_input_focus(self, focus: int, revert_to: int, time: int, onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def get_input_focus(self) -> request.GetInputFocus: ...
+    def query_keymap(self) -> bytes: ...  # TODO: Validate if this is correct
+    def open_font(self, name: str) -> _ResourceBaseClass | None: ...
+    def list_fonts(self, pattern: Pattern[str] | str, max_names: int) -> list[str]: ...
+    def list_fonts_with_info(self, pattern: Pattern[str] | str, max_names: int) -> request.ListFontsWithInfo: ...
+    def set_font_path(self, path: Sequence[str], onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def get_font_path(self) -> list[str]: ...
+    def query_extension(self, name: str) -> request.QueryExtension | None: ...
+    def list_extensions(self) -> list[str]: ...
+    def change_keyboard_mapping(
+        self, first_keycode: int, keysyms: Sequence[Sequence[int]], onerror: ErrorHandler[object] | None = ...
+    ) -> None: ...
+    def get_keyboard_mapping(self, first_keycode: int, count: int) -> list[tuple[int, ...]]: ...
+    def change_keyboard_control(self, onerror: ErrorHandler[object] | None = ..., **keys: object) -> None: ...
+    def get_keyboard_control(self) -> request.GetKeyboardControl: ...
+    def bell(self, percent: int = ..., onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def change_pointer_control(
+        self, accel: tuple[int, int] | None = ..., threshold: int | None = ..., onerror: ErrorHandler[object] | None = ...
+    ) -> None: ...
+    def get_pointer_control(self) -> request.GetPointerControl: ...
+    def set_screen_saver(
+        self, timeout: int, interval: int, prefer_blank: int, allow_exposures: int, onerror: ErrorHandler[object] | None = ...
+    ) -> None: ...
+    def get_screen_saver(self) -> request.GetScreenSaver: ...
+    def change_hosts(
+        self,
+        mode: int,
+        host_family: int,
+        host: Sequence[int] | Sequence[bytes],  # TODO: validate
+        onerror: ErrorHandler[object] | None = ...,
+    ) -> None: ...
+    def list_hosts(self) -> request.ListHosts: ...
+    def set_access_control(self, mode: int, onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def set_close_down_mode(self, mode: int, onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def force_screen_saver(self, mode: int, onerror: ErrorHandler[object] | None = ...) -> None: ...
+    def set_pointer_mapping(self, map: Sequence[int]) -> int: ...
+    def get_pointer_mapping(self) -> list[int]: ...
+    def set_modifier_mapping(
+        self,
+        keycodes: Sequence[Sequence[int]]
+        | tuple[
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+            Sequence[int],
+        ],
```

Comment:
Yes, *but*, it only accepts a very specific number of elements in a sequence. For documentation purposes, this is what I've been doing since python doesn't have fixed-length container types other than tuples.

Reply:
I see — maybe define an `_EightItemIntSequence: TypeAlias = Sequence[int]` alias, instead?

Reply:
This sounds reasonable. Would you like me to do the same for others? (ie define a `_Rectangle` type alias for sequence of 4 ints, _RGB for 3 ints, etc)

Reply:
Sounds like a plan!

Reply:
Alright, trying out some explicitely named type aliases